app-antivirus/clamav-gui/clamav_gui-1.0.9.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Front-End to clamav"
+DESCRIPTION="A tool that simplifies virus scanning on Linux \
+This application provides a graphical user interface for clamav and freshclam with an integrated \
+service menu for Konqueror/Dolphin to scan files directly from your file manager."
+HOMEPAGE="https://github.com/wusel1007/ClamAV-GUI/"
+COPYRIGHT="2015-2025 Joerg Macedo da Costa Zopes"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/specialworld83/archlinux/raw/refs/heads/main/ClamAV-GUI-QT6-$portVersion.tar.gz"
```

Comment:
This needs srcGitRev instead of using the head of main.

Reply:
It's still a wip on this, used the link provided by the AUR recipe, upstream has one but that needs authentication, so not there yet.

Reply:
Fixed also, still have to use the URI from archlinux though.